Makefile

```diff
@@ -18,6 +19,14 @@ ci-run: ## Provisions AWS EC2 hosts for testing staging environment.
 ci-go: ## Creates, provisions, tests, and destroys AWS EC2 hosts for testing staging environment.
 	@if [[ "${CIRCLE_BRANCH}" != docs-* ]]; then molecule test -s aws; else echo Not running on docs branch...; fi
 
+.PHONY: ci-lint-image
+ci-lint-image: ## Builds linting container.
+	docker build $(EXTRA_BUILD_ARGS) -t securedrop-lint:${TAG} -f Dockerfile.linting .
```

Comment:
Please move this `Dockerfile.linting` to a subdirectory. Trying to keep the top-level relatively clean. How about shoving it to `devops/docker` ? ala:

```diff
diff --git a/Dockerfile.linting b/devops/docker/Dockerfile.linting
similarity index 100%
rename from Dockerfile.linting
rename to devops/docker/Dockerfile.linting

diff --git a/Makefile b/Makefile
index 7f9f5700..af2c48b8 100644
--- a/Makefile
+++ b/Makefile
@@ -21,7 +21,7 @@ ci-go: ## Creates, provisions, tests, and destroys AWS EC2 hosts for testing sta

 .PHONY: ci-lint-image
 ci-lint-image: ## Builds linting container.
-       docker build $(EXTRA_BUILD_ARGS) -t securedrop-lint:${TAG} -f Dockerfile.linting .
+       docker build $(EXTRA_BUILD_ARGS) -t securedrop-lint:${TAG} -f devops/docker/Dockerfile.linting .

```